### PR TITLE
feat: [401106, 401107] Amend team name and email

### PR DESCRIPTION
**Pull request by KiranVarma02**
https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/401106

https://dev.azure.com/defragovuk/DEFRA-CDP/_workitems/edit/401107

**BEFORE:**
Screenshot 2024-07-25 at 22 28 18

**MAKE CHANGE**
Screenshot 2024-07-25 at 22 28 44

**CHANGE CONFIRMATION**
Screenshot 2024-07-25 at 22 28 55

Screenshot 2024-07-25 at 22 29 02

### CODE REVIEW

File: designer/server/src/routes/forms/edit.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { redirectWithErrors, schema } from '~/src/routes/forms/create.js'
 
 export const ROUTE_PATH_EDIT_LEAD_ORGANISATION =
   '/library/{slug}/edit/lead-organisation'
+export const ROUTE_PATH_EDIT_TEAM = '/library/{slug}/edit/team'
```
Comment (nayij) [inline designer/server/src/routes/forms/edit.js:11]: teamName or team-name instead of /team perhaps?

**Comment by nayij** [comment]
Are there any tests?

**Review by alexluckett — APPROVED** [review]

File: designer/server/src/routes/forms/edit.test.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      auth
+    }
+
+    const { document } = await renderResponse(server, options)
+
+    const teamName = document.querySelector('#teamName')
+    const teamEmail = document.querySelector('#teamEmail')
+
+    expect(teamName?.value).toBe('Defra Forms')
+    expect(teamEmail?.value).toBe('defraforms@defra.gov.uk')
```
Comment (alexluckett) [inline designer/server/src/routes/forms/edit.test.js:79]: tsc checks:

```
designer/server/src/routes/forms/edit.test.js(78,22): error TS2339: Property 'value' does not exist on type 'Element'.
designer/server/src/routes/forms/edit.test.js(79,23): error TS2339: Property 'value' does not exist on type 'Element'.
```

**Review by alexluckett — COMMENTED** [review]
Thhere

File: designer/server/src/routes/forms/edit.test.js (hunk 2 above)
Comment (KiranVarma02) [reply designer/server/src/routes/forms/edit.test.js:80]: I don't see this error now but there are errors not related to PR. Could you please check?

**Review by alexluckett — APPROVED** [review]

**Pull request merged**